<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trace</class>
 <widget class="QWidget" name="trace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_save">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/icon/Resources/icon/save.png</normaloff>:/icon/Resources/icon/save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_clear">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/icon/Resources/icon/clear.png</normaloff>:/icon/Resources/icon/clear.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_control">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/icon/Resources/icon/pause.png</normaloff>:/icon/Resources/icon/pause.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_delta">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/icon/Resources/icon/flow.png</normaloff>:/icon/Resources/icon/flow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
